README.md

```diff
@@ -26,6 +26,11 @@ This will start up the container and update the NVTs cache - it can take some ti
 
 After that, the scan script will run and the progress will be displayed in the console.
 
+
+#### Target
+
+Target can be a single IP or a comma separated list of IP addresses.
```

Comment:
In previous versions, you could send a 'netmasked' IP like 192.168.1.0/24 and it worked. Did the behavior change?

Reply:
You still should be able to specify a subnet range using CIDR notation. I will update the README to reduce confusion.